<launch>

  <arg name="swap_lr"          default="false"/>
  <arg name="field_of_view"        default="71.86"/>
  <arg name="baseline"    default="7.5"/>

  <node 
    name="depthai_calibration"
    pkg="calibration"
    type="depthai_calibration.py"
    respawn="false"
    output="screen">
    <param name="swap_lr" value="$(arg swap_lr)"/>
    <param name="field_of_view" value="$(arg field_of_view)"/>
    <param name="baseline" value="$(arg baseline)"/>
    <param name="path" value="$(find calibration)"/>

  </node>

</launch>
